docs/GettingStarted.md

```diff
@@ -101,7 +67,7 @@ firstName
 [...]
 ```
 
-The generator has successfully created 100 rows of random data. However, in most cases this data will not be terribly useful. It is likely that `firstName` will only allow a subset of possible string values. If you don't provide any constraints, the generator will output random strings containing basic latin characters and punctuation.
+The generator has successfully created 100 rows of random data. However, in most cases this data will not be terribly useful. It is likely that `firstName` should only allow a subset of possible string values. If you don't provide any constraints, the generator will output random strings containing basic latin characters and punctuation.
```

Comment:
The third sentence confused me a little - I would rephrase it along the lines of "If you have a field named `firstName`, it is likely that you want it to only contain strings that resemble real names to some degree.  However, if you don't..."

Reply:
I agree, I have changed the guide to follow through developing an example a bit more strongly.

This section has been rewritten as a result